src/resources/views/signup.ejs

```diff
@@ -61,20 +60,31 @@
   </script>
   <%- include('./partials/gtag'); %>
     <script>
-      function togglePassword() {
-        var x = document.getElementById("app-input-password")
-        var y = document.getElementById("icon-show-password")
-        const type = x.type === "password" ? "text" : "password"
-        x.type = type
-        y.classList.contains("fa-eye-slash") ? y.classList.replace("fa-eye-slash", "fa-eye") : y.classList.replace("fa-eye", "fa-eye-slash")
-      }
-      function togglePasswordConfirm() {
-        var x = document.getElementById("app-input-password-confirm")
-        var y = document.getElementById("icon-show-password-confirm")
-        const type = x.type === "password" ? "text" : "password"
-        x.type = type
-        y.classList.contains("fa-eye-slash") ? y.classList.replace("fa-eye-slash", "fa-eye") : y.classList.replace("fa-eye", "fa-eye-slash")
-      }
+      $(document).ready(function () {
+        var password = $("#app-input-password")
+        $('#icon-show-password').click(function () {
+          if (password.prop("type") == "password") {
+            $(this).removeClass("fa-eye-slash")
+            password.attr("type", "text")
+          }
+          else {
+            $(this).addClass("fa-eye-slash")
+            password.attr("type", "password")
+          }
+        })
```

Comment:
var password = $("#app-input-password")
var isPasswordVisible = password.prop("type") === "text";
$('#icon-show-password').click(function () {
   $(this).toggleClass("fa-eye-slash", isPasswordVisible)
   password.attr("type", isPasswordVisible ? "password" : "text")
})